plugins/lang-plan-a/src/main/java/org/elasticsearch/plan/a/PlanAParserBaseVisitor.java

```diff
@@ -0,0 +1,357 @@
+// ANTLR GENERATED CODE: DO NOT EDIT
+package org.elasticsearch.plan.a;
```

Comment:
> So that IDEs work and other reasons. Its explained in the build if you take the time to read it.

Thanks @rmuir . That's all the was asked (though, I read the build.gradle file, couldn't figure it out from there).

I don't think you should conflate questions/suggestions as "you should do something different". A short explanation and polite answer would be great.